tests/lib/rook/upstream.py

```diff
@@ -32,12 +32,8 @@ def __init__(self, workspace, kubernetes):
         self.build_dir = os.path.join(self.workspace.build_dir, 'rook')
         self.ceph_dir = os.path.join(
             self.build_dir, 'cluster/examples/kubernetes/ceph')
-
-    def preinstall(self):
-        super().preinstall()
-        if converter('@bool', settings.UPSTREAM_ROOK.BUILD_ROOK_FROM_GIT):
-            self.upload_rook_image()
-            self._fix_yaml()
+        self.rook_chart = settings(
+            f"{settings.UPSTREAM_ROOK}.rook_ceph_chart")
```

Comment:
This should probably just bet settings.UPSTREAM_ROOT.ROOK_CEPH_CHART

Reply:
Fixed